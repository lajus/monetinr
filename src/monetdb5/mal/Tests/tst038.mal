#test the capability to deal with unspecified bats.
function tst2(b:bat[:oid,:any_1]);
	bat.append(b,"hello");
end tst2;

b:= bat.new(:oid,:int);

#The call to tst2 fails at runtime
user.tst2(b);
io.print("shouldn't come here\n");
