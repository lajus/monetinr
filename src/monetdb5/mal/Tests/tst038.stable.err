stderr of test 'tst038` in directory 'monetdb5/mal` itself:


# 18:26:03 >  
# 18:26:03 >  Mtimeout -timeout 60 Mserver "--config=/ufs/mk/monet5/Linux/etc/monet.conf" --debug=10 --set "monet_mod_path=/ufs/mk/monet5/Linux/lib/MonetDB" --set "gdk_dbfarm=/ufs/mk/monet5/Linux/var/MonetDB/dbfarm" --set "sql_logdir=/ufs/mk/monet5/Linux/var/MonetDB/log" --set mapi_port=49025 --set sql_port=56914 --set monet_prompt= --trace "--config=/ufs/mk/monet5/src/mal/Tests/All.conf" --dbname=mTests_src_mal  tst038.mal </dev/null
# 18:26:03 >  


# 18:26:04 >  
# 18:26:04 >  Done.
# 18:26:04 >  

