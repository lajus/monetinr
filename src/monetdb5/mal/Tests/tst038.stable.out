stdout of test 'tst038` in directory 'monetdb5/mal` itself:


# 07:58:43 >  
# 07:58:43 >  Mtimeout -timeout 60 Mserver "--config=/ufs/mk/monet5/Linux/etc/MonetDB5.conf" --debug=10 --set "monet_mod_path=/ufs/mk/monet5/Linux/lib/MonetDB5:/ufs/mk/opensource/MonetDB/Linux/lib/MonetDB" --set "gdk_dbfarm=/ufs/mk/monet5/Linux/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/mk/monet5/Linux/var/MonetDB5/log" --set mapi_port=42158 --set sql_port=55708 --set monet_prompt= --trace "--config=/ufs/mk/monet5/src/mal/Tests/All.conf" --dbname=mTests_src_mal  tst038.mal </dev/null
# 07:58:43 >  

# MonetDB 5 server v11.12.0
# This is an unreleased version
# Serving database 'mTests_mal', using 8 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Found 15.629 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2013 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://vienna.ins.cwi.nl:32160/
# MonetDB/GIS module loaded
# MonetDB/JAQL module loaded
# MonetDB/SQL module loaded
# MonetDB/DataCell loaded
function user.tst2(b:bat[:oid,:any_1]):void;
    bat.append(b,"hello");
end tst2;
!TypeException:user.tst2[1]:'bat.append' undefined in: _2:bat[:oid,:str] := bat.append(b:bat[:oid,:int], _3:str)
!MALException:user.tst2[0]:Error in cloned function
!TypeException:user.main[4]:'user.tst2' undefined in: _6:void := user.tst2(b:bat[:oid,:int])
function user.main():void;
#test the capability to deal with unspecified bats. 
    b := bat.new(:oid,:int);
#The call to tst2 fails at runtime 
    user.tst2(b);
    io.print("shouldn't come here\n");
end main;

# 07:58:43 >  
# 07:58:43 >  Done.
# 07:58:43 >  

